### PR TITLE
Cleans up lodash import

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

**Comment by stefanpenner** [comment]
👍

File: package.json [hunk 1]
```diff
@@ -22,7 +22,9 @@
     "broccoli-merge-trees": "^1.0.0",
     "debug": "^2.2.0",
     "exists-sync": "0.0.3",
-    "lodash": "^4.5.0",
+    "lodash.assign": "^4.1.0",
```
Comment (bmeurant) [inline package.json]: ``lodash.assign`` should not be declared in ``devDependencies`` ?

**Comment by Turbo87** [comment]
Should we use our ember-cli-lodash-subset library here instead? Seems much more efficient than those lodash packages with all their subdependencies

**Comment by locks** [comment]
@Turbo87 I'm unfamiliar with the workings of that package, I'll wait for your decision.

**Comment by Turbo87** [comment]
@locks looks like they are using rollup or something similar for those packages now too and have ditched the subdependencies. should be fine to use either.

just for reference:

```
 52K	./node_modules/ember-cli-lodash-subset
 32K	./node_modules/lodash.assign
 80K	./node_modules/lodash.find
 36K	./node_modules/lodash.uniq
```

which can be explained by `ember-cli-lodash-subset` having minified code

File: package.json [hunk 2]
```diff
@@ -22,12 +22,14 @@
...
+    "lodash.find": "^4.5.1",
+    "lodash.uniq": "^4.4.0",
     "process-relative-require": "^1.0.0",
     "silent-error": "^1.0.0"
   },
   "devDependencies": {
     "chai": "^3.0.0",
-    "mocha": "^3.0.1"
+    "mocha": "^3.0.1",
+    "lodash.assign": "^4.1.0"
```
Comment (Turbo87) [inline package.json]: nitpicking here, but alphabetic sorting will reduce future diffs when `npm install` is used 😉

Comment (stefanpenner) [inline package.json]: lets fix then, then :shipit:

Comment (locks) [inline package.json]: 👍 ?

**Comment by stefanpenner** [comment]
> 52K    ./node_modules/ember-cli-lodash-subset

let got for ^ if it already includes the required pieces.

**Pull request merged**